cleanlab/internal/util.py

```diff
@@ -430,54 +430,3 @@ def print_joint_matrix(joint_matrix, round_places=2):
         short_title="p(labels,y)",
         round_places=round_places,
     )
-
-
-def _python_version_is_compatible(
-    warning_str="pyTorch supports Python version 3.6, 3.7, 3.8, 3.9",
-    warning_already_issued=False,
-    list_of_compatible_versions=[3.5, 3.6, 3.7, 3.8],
-):
-    """Helper function for VersionWarning class that issues
-    a warning (if a warning has not already been issued),
-    whenever the python version is not in the
-    list_of_compatible_versions.
-    """
-
-    import sys
-
-    v = sys.version_info[0] + 0.1 * sys.version_info[1]
-    if v in list_of_compatible_versions:
-        return True
-    elif not warning_already_issued:  # pragma: no cover
-        import warnings
-
-        warning = """
-        {}
-        cleanlab supports Python >= 3.6, but you are using Python version {}.
-        You'll need to use a version compatible with both.""".format(
-            warning_str, v
-        )
-        warnings.warn(warning)
-        return False
-
-
-class VersionWarning(object):
```

Comment:
Effectively we are saying that we will support all versions of python 3.6 and up and if you use pytorch or external libraries that don't support a version of python that cleanlab, supports, this will fail and you'll get a generic warning that you need to figure out. Overall, seems okay given that most libraries we interact with support similar versions than we do. It certainly makes the code a lot cleaner :)